docs/_guide/decorators.md

```diff
@@ -0,0 +1,260 @@
+
+```js
+import {LitElement, html, customElement, property} from 'lit-element';
+
+@customElement('my-element')
+class MyElement extends LitElement {
+
+ // Declare observed properties
+ @property()
+ adjective = 'awesome';
+
+ // Define the element's template
+ render() {
+   return html`<p>your ${adjective} template here</p>`;
+ }
+}
+```
+
+## Enabling decorators
+
+To use decorators, you need to use a compiler such as Babel or the TypeScript compiler.
+
+<div class="alert alert-info">
+
+**The decorators proposal**. Decorators are a stage 2 proposal for addition to the ECMAScript standard, which means they're not implemented in browsers yet. Compilers like Babel provide support for proposed features like decorators by compiling them into JavaScript a browser can run.
+
+</div>
+
+### To use decorators with TypeScript
+
+To use decorators with TypeScript, enable the `experimentalDecorators` compiler option.
+
+```json
+"experimentalDecorators": true,
+```
+
+Enabling `emitDecoratorMetadata` is not required and not recommended.
+
+### To use decorators with JavaScript
+
+If you're  JavaScript with Babel, you can enable decorators by adding  the following plugins:
+
+*   [`@babel/plugin-proposal-decorators`](https://babeljs.io/docs/en/babel-plugin-proposal-decorators). 
+*   [`@babel/plugin-proposal-class-properties`](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties)
+
+To enable the plugins, you'd add code like this to your Babel configuration:
+
+```js
+plugins = [
+  '@babel/plugin-proposal-class-properties',
+  ['@babel/proposal-decorators', {decoratorsBeforeExport: true}],
+];
+```
+
+## LitElement decorators
+
+LitElement provides the following decorators:
+
+*   <code>[customElement](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#customelement)</code>. Define a custom element.
+*   <code>[eventOptions](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#eventoptions)</code>. Add event listener options to a listener added in your template.
+*   <code>[property](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#property)</code> and <code>[internalProperty](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#internalproperty)</code>. Define properties.
+*   <code>[query](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#query)</code>, <code>[queryAll](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#queryAll)</code>, and <code>[queryAsync](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#queryAsync). </code>Create a property getter that returns specific elements from your component's render root.
+*   <code>[queryAssignedNodes](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#queryAssignedNodes)</code>. Create a property getter that returns the children assigned to a specific slot. 
```

Comment:
`queryAssignedNodes ` link doesn't work.

Reply:
Should be fixed now...